internal/tuf/tuf.go

```diff
@@ -82,10 +82,11 @@ type Role struct {
 
 // RootMetadata defines the schema of TUF's Root role.
 type RootMetadata struct {
-	Type    string          `json:"type"`
-	Expires string          `json:"expires"`
-	Keys    map[string]*Key `json:"keys"`
-	Roles   map[string]Role `json:"roles"`
+	Type                   string          `json:"type"`
+	Expires                string          `json:"expires"`
+	Keys                   map[string]*Key `json:"keys"`
+	Roles                  map[string]Role `json:"roles"`
+	GitHubApprovalsTrusted bool            `json:"githubApprovalsTrusted"`
```

Comment:
I lean towards keeping it vendor specific to start because the attestations we generate may differ. I wonder if the approvers predicate here should be accompanied by a github PR attestation, for example.